test/resolver/ecto_test.exs

```diff
@@ -0,0 +1,99 @@
+defmodule Resolver.EctoTest do
+  use ExUnit.Case
+  alias Backoffice.Resolvers.Ecto, as: EctoResolver
+  require Ecto.Query
+
+  defmodule Todo do
+    use Ecto.Schema
+
+    @primary_key false
+    schema "todos" do
+      field(:userId, :string)
+      field(:id, :string)
+      field(:completed, :boolean)
+      field(:title, :string)
+    end
+  end
+
+  defmodule MockRepo do
```

Comment:
I looked into Mox but backed away because I misunderstood the docs to require a concrete type for the module being replaced. This is definitely better than what I was doing.